pages/nft-book-store/user/index.vue

```diff
@@ -128,6 +128,14 @@
             size="lg"
             @click="onLoginToStripe"
           />
+          <UButton
+            v-else-if="!userLikerInfo?.user"
+            label="Liker ID is required"
```

Comment:
maybe **Register Liker ID to Continue** or **Complete Liker ID Registration** ?

Reply:
Maybe just `Setup your ID` like above